saleor/graphql/schema.graphql

```diff
@@ -317,23 +316,25 @@ type AppExtensionCountableEdge {
 }
 
 input AppExtensionFilterInput {
-  view: AppExtensionViewEnum
-  type: AppExtensionTypeEnum
+  mount: [AppExtensionMountEnum]
   target: AppExtensionTargetEnum
 }
 
-enum AppExtensionTargetEnum {
-  MORE_ACTIONS
-  CREATE
-}
-
-enum AppExtensionTypeEnum {
-  OVERVIEW
-  DETAILS
+enum AppExtensionMountEnum {
+  PRODUCT_OVERVIEW_CREATE
+  PRODUCT_OVERVIEW_MORE_ACTIONS
+  PRODUCT_DETAILS_MORE_ACTIONS
+  NAVIGATION_CATALOG
+  NAVIGATION_ORDERS
+  NAVIGATION_CUSTOMERS
+  NAVIGATION_DISCOUNTS
+  NAVIGATION_TRANSLATIONS
+  NAVIGATION_PAGES
 }
 
-enum AppExtensionViewEnum {
-  PRODUCT
+enum AppExtensionTargetEnum {
+  POPUP
+  APP_PAGE
```

Comment:
Was it called "Open As" before and now we changed it to target? I think "open as" is a better name for this setting.

Reply:
That was a suggestion added by @jwm0  as he was handling the concept of changes. He said that `target` is popular according to popups

Reply:
OK, if this is approved by frontend then I'm fine with this 👍